src/platforms/android/common/proguard.mdx

```diff
@@ -6,6 +6,8 @@ description: "Learn about using the Sentry Gradle Plugin or sentry-cli to upload
 
 If you want to see de-obfuscated stack traces, you'll need to use [ProGuard](https://www.guardsquare.com/proguard) with Sentry. To do so, upload the ProGuard mapping files by either the recommended method of using our [Gradle integration](/platforms/android/gradle/) or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload).
 
+But either way, it works the same way. A (random) UUID will be or has to be generated and placed into the `AndroidManifest.xml`. The same UUID has to be used to upload the mapping file(s). In case a crash happens in the app, the Sentry Android SDK will, along with the stacktrace, send the UUID to the Sentry server instance. Sentry can now de-obfuscate the stacktrace by searching for the associated mapping file with the same UUID.
+
```

Comment:
```suggestion
It works the same either way. A (random) UUID will be or has to be generated and placed into the `AndroidManifest.xml`. The same UUID has to be used to upload the mapping file(s). In case a crash happens in the app, the Sentry Android SDK will send the UUID along with the stacktrace to the Sentry server instance. Sentry will then de-obfuscate the stacktrace by searching for the associated mapping file with the same UUID.

```